src/CommunityToolkit.Maui.UnitTests/Mocks/AnimationExtensions.cs

```diff
@@ -72,7 +72,7 @@ public override async void Start()
 					}
 					catch (Exception e)
 					{
-						Trace.WriteLine(e);
+						Trace.TraceError(e.StackTrace);
```

Comment:
I think we should rely on the full informartion

```suggestion
						Trace.TraceError(e);
```